backend/django/create-user-model.mo.md

```diff
@@ -0,0 +1,74 @@
+# Context:
+You have a Django project installed. You need to handle different types of users
+
+# Prerequisite:
+Have the project installed.
+https://github.com/bamlab/dev-standards/blob/master/backend/django/getting-started.mo.md
+
+# Situation:
+You have a basic user model with a name.
```

Comment:
Checklist please. It render to 
![image](https://user-images.githubusercontent.com/13785185/32285665-4076ccbc-bf2b-11e7-83d3-4d9ac2c26ae9.png)